workflow/scripts/_joint_scanpy_analysis/embed_condition_plot.py

```
@@ -0,0 +1,224 @@
+obs["cell_id"] = obs.index
+
+# fix name for proseg
+if "proseg" in segmentation.stem:
+    obs["segmentation"] = segmentation.stem
+
+
+if color == "sample":
+    # plot sample as color, no need to load annotations
+    df = obs
+    params = color
+    title = f"Segmentation: {segmentation.stem}, condition: {condition.stem}, Panel: {panel.stem}"
```

Ruff (0.14.8)

Undefined name `panel`

(F821)